src/main/java/frc/robot/subsystems/drivetrain/SwerveModule.java

```diff
@@ -52,6 +52,8 @@ public class SwerveModule implements BaseSwerveModule {
     private static final double steerD = 0;
     private static final double steerFF = 0;
 
+    private boolean VERBOSE_SHUFFLEBOARD;
```

Comment:
I don't think this should be a constructor argument? Could just set it as a flag in either here or `Constants.java` so that you don't need to pass it around everywhere.